example/go.mod

```diff
@@ -1,11 +1,12 @@
 module github.com/google/go-github/v69/example
 
-go 1.23.0
+go 1.23.1
```

Comment:
Doesn't this work with `go 1.23.0` ?
We would prefer to use that due to https://github.com/google/go-github/pull/3423#issuecomment-2577882152

Reply:
no problem at all, fixed.
I just misunderstood your previous comment to mean that you want line 3 to remain.